### PR TITLE
other-functions.md example fix

### PR DESCRIPTION
**Pull request by alexeyerm**
### Changelog category:

- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/sql-reference/functions/other-functions.md [hunk 1]
```diff
@@ -571,7 +571,7 @@ Example:
 
 ``` sql
 SELECT
-    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vk.com'], ['www.yandex', 'example.com']) AS s,
+    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vkontakte.ru'], ['www.yandex', 'example.com', 'vk.com'] AS s,
```
Comment (nickitat) [inline docs/en/sql-reference/functions/other-functions.md:574]: you lost closing bracket of `transform()`

Comment (alexeyerm) [reply docs/en/sql-reference/functions/other-functions.md:574]: Omfg...
Will fix now.

Comment (alexeyerm) [inline docs/en/sql-reference/functions/other-functions.md:574]:
```suggestion
    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vkontakte.ru'], ['www.yandex', 'example.com', 'vk.com']) AS s,
```

File: docs/zh/sql-reference/functions/other-functions.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ ORDER BY c DESC
 
 ``` sql
 SELECT
-    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vk.com'], ['www.yandex', 'example.com']) AS s,
+    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vkontakte.ru'], ['www.yandex', 'example.com', 'vk.com'] AS s,
```
Comment (alexeyerm) [inline docs/zh/sql-reference/functions/other-functions.md:240]:
```suggestion
    transform(domain(Referer), ['yandex.ru', 'google.ru', 'vkontakte.ru'], ['www.yandex', 'example.com', 'vk.com']) AS s,
```

File: docs/en/sql-reference/functions/other-functions.md (hunk 1 above)
Comment (alexeyerm) [reply docs/en/sql-reference/functions/other-functions.md]: Thanks.
My first web GITing experience... :)

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**